server.js

```diff
@@ -29,7 +60,47 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Music makes people come togheter!')
+})
+
+// Using Async + Await
+// app.get('/topmusics', async (req, res) => {
+//   const topmusic = await TopMusic.find()
+//   res.json(topmusic)
+//   })
+
+//Using promises
+app.get('/topmusics', (req, res) => {
+  TopMusic.find()
+  .then((results) => {
+    // console.log('Found: ' + results)
+    res.json(results)
+  })
+})
+
+app.get('/topmusics/id/:id', (req, res) => {
+  const id = req.params.id
+  TopMusic.findOne({'id': id})
+  .then((results) => {
+    // console.log('Found: ' + results)
+    if (results === null)
+    throw "wrong" 
+    res.json(results)
+  }).catch((err) => {
+    console.log('Error' + err)
+    res.json({message: 'Cannot find this song', err: err})
```

Comment:
Consider changing the message to something like "Could not complete request" and let `err` explain more about why.